api/types.go

```diff
@@ -538,7 +505,7 @@ func DefaultOptions() Options {
 			LowVRAM:   false,
 			F16KV:     true,
 			UseMLock:  false,
-			UseMMap:   TriStateUndefined,
+			UseMMap:   nil,
```

Comment:
I think we can keep this `true` (if we want it on by default that is). Otherwise we can just omit this from the struct as it will default to `nil`

Reply:
We need the default to be unset in order to detect if the client actually passes in a true or false, or just leaves it blank.  Otherwise we can't have the [use-case specific default behavior logic](https://github.com/ollama/ollama/blob/main/llm/server.go#L219-L227)  As to the pedantic nil, yes, we could omit it, but since it's treated as a bool largely, I thought it good to be explicit so it's not ambiguous as a false for better readability of the code.